department-of-veterans-affairs/va.gov-cms
# VACMS-9171 announcement block & alert updates

## Description
Closes #9171. Had a design sync with @cjterdi and @rachel-kauff and we collaboratively updated some of the alert styles in concert with the new CMS announcement block type.

- Created new announcement block component in the design system w/ interactive storybook example for the different types (maybe should be split out to 4 standalone like the other alerts are)
- Updated existing theme templates for announcement blocks to use the DS
- Updated system status alerts to match new design direction

## Testing done
- New design system component in storybook
- Added a new announcement block to specific content types. Verified that all 4 types of announcement block render correctly
- Verified new alert styles render correctly

## Screenshots
<img width="1263" alt="Screen Shot 2022-05-26 at 1 39 17 PM" src="https://user-images.githubusercontent.com/11279744/170576533-9a43175f-3ce2-47ce-82df-764c30902f3b.png">
<img width="1439" alt="Screen Shot 2022-05-26 at 1 40 45 PM" src="https://user-images.githubusercontent.com/11279744/170576540-75906ec7-543c-46b1-baad-bb5c8f8ae85f.png">


## QA steps

- Visit the storybook link and see that there is a new component for the announcement block
- In storybook see that existing alerts now have a variant for no background (you may need to hit 'a' to bring up the controls panel)

- In Drupal, log in as an admin and visit `/block/add?destination=/admin/structure/block/block-content`.
- Add a cms announcement. Block description is the administrative label that will appear when it is placed in the block layout list, the title field is the title of the announcement.
- After creating the block, visit `/admin/structure/block` and place this new block in the Content Region (search for your label to find it quickly). Configure it to only be visible on certain content types (ex. VAMC Facility)

<img width="706" alt="Screen Shot 2022-05-26 at 1 48 47 PM" src="https://user-images.githubusercontent.com/11279744/170577691-7a46b42d-e45c-4686-814b-876bf223a1f1.png">
- After placing the block, drag it to the top of the region to ensure it shows up where it's supposed to on the page. 

- Visit an existing node of the content type you restricted the block visibility to. See that hte block shows up on the page, and system status alerts appear below it with the updated styles.

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
